index.py

```diff
@@ -10,10 +10,11 @@
 from rnaseqc.over_time import layout as rnaseqc_overtime_layout
 from runreport.proj_hist import layout as runreport_projhist_layout
 from runscanner.yield_over_time import layout as runscanner_yield_over_time_layout
+from pooling_qc_sample import layout as pooling_qc_layout
```

Comment:
`pooling_qc_sample` -> `poolqc.pooling_qc_sample` (`name_of_directory.name_of_file`)

Reply:
Please update this.